sync_diff_inspector/diff.go

```diff
@@ -322,6 +323,17 @@ func (df *Diff) compareStruct(ctx context.Context, tableIndex int) (isEqual bool
 	return isEqual, isSkip, nil
 }
 
+func verifySnap(latestSnap []string) bool {
+	if len(latestSnap) != 1 {
+		return false
+	}
+	val, err := strconv.ParseInt(latestSnap[0], 10, 64)
```

Comment:
🤔 

As `latestSnap` is a string, if the two timestamps are lexicographically reversed the `strings.Compare() < 0` below will also be wrong. Although this won't be a problem until 2090-11-18 22:07:45.625